backend/src/main/java/gov/cdc/usds/simplereport/logging/RestLoggingInterceptor.java

```diff
@@ -24,8 +38,23 @@ public boolean preHandle(
         request.getMethod(),
         sanitizeRequestURI(request.getRequestURI()),
         handler);
-    String requestId = UUID.randomUUID().toString();
+    Organization org = null;
+    ApiUser userInfo = null;
+    try {
+      org = organizationService.getCurrentOrganization();
+      userInfo =
+          apiUserContextHolder.hasBeenPopulated() ? apiUserContextHolder.getCurrentApiUser() : null;
+    } catch (NoSuchElementException | MisconfiguredUserException e) {
+      // account for rest endpoints that are hit without okta info
+      log.debug("Exception getting additional logging context.", e);
```

Comment:
Also I haven't really used trace at all - is the intention just to make it visible / enabled for production logging?

Reply:
yea, my concern would just be too much noise, not sure if its actionable, it would throw for patients checking on their results since they are not logged in, is that correct?

is there a case where you might have a user without an org or vice versa? does it make sense to separate these two try blocks

Reply:
I don't think you could hit this without either both or neither.